src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -166,6 +174,19 @@ def _check_conditional_args_embeddings(
                 "The --batch-size option is currently only supported with the embeddings endpoint type."
```

Comment:
It looks like you're using batch size for rankings, which is awesome. I think all of the references to batch-size need to be updated to state that the ranking endpoint is also supported (e.g. here, in the README, in the code comments in LLM inputs, etc.). You should be able to search for "batch" in this folder and find all of the references to update.